scipy/integrate/quadpack.py

```diff
@@ -633,6 +633,18 @@ def tplquad(func, a, b, gfun, hfun, qfun, rfun, args=(), epsabs=1.49e-8,
     odeint: ODE integrators
     scipy.special: For coefficients and roots of orthogonal polynomials
 
+    Examples
+    --------
+
+    Returns triple integration of ``x*y*z``
+    x ranging from 1 to 2,y ranging from 2 to 3,z ranging from 0 to 1
+
+    >>> from scipy import integrate
+    >>> f=lambda z,y,x: x*y*z
+    >>> integrate.tplquad(f,1,2,lambda x: 2,lambda x: 3,lambda x,y: 0,lambda x,y: 1)
```

Comment:
spaces after commas. If line goes over 80 chars, then carry over to next line (no \ required)